src/main/java/dev/enjarai/rollingdowninthedeep/RollingDownInTheDeep.java

```diff
@@ -19,32 +19,29 @@
 
 public class RollingDownInTheDeep implements ModInitializer {
 	public static final String MOD_ID = "rolling_down_in_the_deep";
-	public static final Logger LOGGER = ProperLogger.getLogger(MOD_ID);
-	public static final Sensitivity SMOOTHING = new Sensitivity(5, 5, 1);
-
+	//	public static final Logger LOGGER = ProperLogger.getLogger(MOD_ID);
+	public static final Sensitivity SMOOTHING = new Sensitivity(.02, .02, .02);
 	public static final RollGroup SWIM_GROUP = RollGroup.of(id("swimming"));
 	public static final RollGroup DABR_GROUP = RollGroup.of(new Identifier("do_a_barrel_roll", "fall_flying"));
 
 	@Override
 	public void onInitialize() {
 		SWIM_GROUP.trueIf(RollingDownInTheDeep::shouldRoll);
 
-//		RollEvents.EARLY_CAMERA_MODIFIERS.register(context -> context
-//				.useModifier(RotationModifiers.strafeButtons(1800)), // hahha yes, we dont need this do we? i sure hope we dont
-//				,
-//				10, () -> SWIM_GROUP.get() && !DABR_GROUP.get());
-
+		// Smooth the camera rotation
 		RollEvents.EARLY_CAMERA_MODIFIERS.register(context -> context
-						.useModifier(ModConfig.INSTANCE::configureRotation),
+				.useModifier(CameraModifiers::smoothCamera), 1000, () -> SWIM_GROUP.get() && !DABR_GROUP.get());
+
+		RollEvents.EARLY_CAMERA_MODIFIERS.register(context -> context.useModifier(StrafeRollModifiers::applyStrafeRoll),
+				1000, () -> SWIM_GROUP.get() && !DABR_GROUP.get());
+
+		RollEvents.EARLY_CAMERA_MODIFIERS.register(context -> context.useModifier(ModConfig.INSTANCE::configureRotation),
 				1000, () -> SWIM_GROUP.get() && !DABR_GROUP.get());
```

Comment:
When registering multiple modifiers on the same roll event, it'd be better to chain multiple `useModifier` calls on the `context` object. Alternatively, you could set the priority for each one depending on their desired apply order.

If you just have it like this, the order in which they apply is arbitrary, which is usually not a good idea.